### PR TITLE
resource/alicloud_click_house_db_cluster: Fixed value of `payment_type` from `Prepay` to `Prepaid`

### PR DESCRIPTION
**Pull request by MrWolong**

### CODE REVIEW

File: alicloud/resource_alicloud_click_house_db_cluster_test.go [hunk 1]
```diff
@@ -368,6 +368,66 @@ func TestAccAlicloudClickHouseDBCluster_basic2(t *testing.T) {
...
+	name := fmt.Sprintf("tf-testacc%sclic  khousedbcluster%d", defaultRegionToTest, rand)
+	testAccConfig := resourceTestAccConfigFunc(resourceId, name, AlicloudClickHouseDBClusterBasicDependence0)
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+			testAccPreCheckWithRegions(t, true, connectivity.ClickHouseSupportRegions)
+		},
+		IDRefreshName: resourceId,
+		Providers:     testAccProviders,
+		//CheckDestroy:  rac.checkResourceDestroy(),
```
Comment (xinmu0426) [inline alicloud/resource_alicloud_click_house_db_cluster_test.go:390]: CheckDestroy: nil

Comment (xinmu0426) [reply alicloud/resource_alicloud_click_house_db_cluster_test.go:390]: 另外针对预付费实例，增加每个月只跑一次的限制

File: alicloud/resource_alicloud_click_house_db_cluster.go [hunk 2]
```diff
@@ -501,15 +504,16 @@ func convertClickHouseDbClusterPaymentTypeRequest(source string) string {
 	case "PayAsYouGo":
 		return "Postpaid"
 	case "Subscription":
-		return "Prepay"
+		return "Prepaid"
```
Comment (xinmu0426) [inline alicloud/resource_alicloud_click_house_db_cluster.go:507]: 这个是不兼容变更啊，这个有aone么

Comment (MrWolong) [reply alicloud/resource_alicloud_click_house_db_cluster.go:507]: https://aone.alibaba-inc.com/req/42919843?from=ak&spm=a2o8d.corp_prod_workitem_my.0.0.a59c7e55lReUDR

File: alicloud/resource_alicloud_click_house_db_cluster_test.go (hunk 1 above)
Comment (MrWolong) [reply alicloud/resource_alicloud_click_house_db_cluster_test.go]: 已修复

**Pull request merged**